codalab/server/rest_server.py

```diff
@@ -48,6 +50,11 @@
 
 logger = logging.getLogger(__name__)
 
+sentry_sdk.init(
+    dsn="https://a9654419762745c3b255bf299f28a656@o431750.ingest.sentry.io/5383515",
```

Comment:
We shouldn't hard code this URL. Instead, we should use the CODALAB_SENTRY_INGEST_URL environment variable.